web/static/js/components/thesis-editor.js

```diff
@@ -279,20 +253,20 @@ class ThesisEditor extends React.Component {
       el.classList.remove('thesis-tray-open')
     }
 
-    document.title = this.state.title
-    this.pageDescriptionMetaTag().content = this.state.description
-    thesisContainer.setAttribute('data-redirect-url', this.state.redirectURL)
+    this.external.setTitle(this.state.title)
+    this.external.setDescription(this.state.description)
+    this.external.setRedirectURL(this.state.redirectURL)
   }
 
-  dynamicEnabled () {
-    return this.state.templates.length > 0
+  canCreatePages () {
```

Comment:
Renamed this because `dynamicEnabled` was confusing me. `canCreatePages` makes more sense to me.